app/javascript/react/screens/App/Overview/components/Cards/MigrationsCompletedCard/MigrationCompletedRow.js

```diff
@@ -1,20 +1,23 @@
 import moment from 'moment';
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Popover } from 'react-bootstrap';
```

Comment:
this one is already exposed in PF-React, no?

Reply:
ah yes...

Reply:
Fixed.